### ISSUE TITLE
Custom styles for checkbox

### ISSUE THREAD

**Issue by btzr-io** [btzr-io/Villain#137]
Add a custom style for checkbox component on the demo example.
> Note: it should be simple ( minimalist ) and not affect usability or accessibility
